### ISSUE TITLE
Add an option to hide icon

### ISSUE THREAD

**Issue by RomainCscn** [nbolar/PlayStatus#31]
Hello, do you plan to add an option to hide the icon? I'd love to be able to do it.

Thanks!

**Comment by nbolar** [comment]
Hi @RomainCscn, So if there is no music playing and the icon is hidden, it would be very hard to find where the app resides on the menu bar. 

With that being said let me play around with this idea...

**Comment by RomainCscn** [comment]
Yeah, I meant when playing, I found the animating icon distracting. 
I think it's ok to leave it when no music is being played.

**Comment by nbolar** [comment]
Animating icon? The icon itself does not animate. There are a couple of other animations that happen depending on your settings:
 - The slide in title on new song can be disabled in the settings.
 - The scrollable title option can also be disabled.

If the icon itself is animating, then it's a bug.

**Comment by RomainCscn** [comment]
Sorry, I was confused by another project. Just hiding the icon while playing would be perfect.

**Comment by abraver** [comment]
+1 — I'd love this feature!
